tests/lib/parse.js

```diff
@@ -82,4 +82,24 @@ describe("parse()", () => {
         });
 
     });
+
+    describe("acorn", () => {
+        it("acorn.parse() should not re-use the same identifier node in object patterns", () => {
```

Comment:
```suggestion
        it("should not re-use the same identifier node in shorthand properties", () => {
```

since it's also testing object expressions